Sources/CNIOSHA1/c_nio_sha1.c

```diff
@@ -58,7 +58,7 @@
 
 
 
-/* sanity check */
+/* soundness check */
```

Comment:
this change needs to be done in `update_and_patch_sha1.sh`. This file can't be manually edited because it's vendored by that script. Also add it to the top-level comment which summarises the changes.